bokehjs/src/coffee/models/annotations/color_bar.coffee

```diff
@@ -46,6 +46,9 @@ class ColorBarView extends Annotation.View
   _set_canvas_image: () ->
     palette = @model.color_mapper.palette
 
+    if @model.orientation == 'vertical'
+      palette = palette.slice(0).reverse()
```

Comment:
There must be a better way of expressing an intent to create a copy of an array in a more readable way. Not blaming the author, because I've been using the same idiom myself. I looked at underscore.js, but its documentation says there's no appropriate function, yet the implementation says something different. One can use `_.clone()` for this.